soc/bouffalolab/bl6/soc.c

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright (c) 2021-2024 ATL Electronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Bouffalo Lab RISC-V MCU series initialization code
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/init.h>
+#include <zephyr/irq.h>
+
+#include <bflb_hbn.h>
+#include <bflb_glb.h>
+#include <clic.h>
+
+/* clang-format off */
```

Comment:
???

Reply:
We use many configs when accessing registers directly. clang-format not helps to keep code easy to understand and this is why we disable it.